core/sis-cql/src/main/java/org/apache/sis/cql/CQL.java

```diff
@@ -43,21 +49,23 @@
 import org.locationtech.jts.geom.LineString;
 import org.locationtech.jts.geom.LinearRing;
 import org.locationtech.jts.geom.Polygon;
-import org.opengis.filter.And;
+import org.opengis.util.FactoryException;
+import org.opengis.feature.Feature;
+import org.opengis.filter.Expression;
 import org.opengis.filter.Filter;
-import org.opengis.filter.FilterFactory2;
-import org.opengis.filter.Or;
-import org.opengis.filter.expression.Expression;
-import org.opengis.filter.expression.PropertyName;
+import org.opengis.filter.FilterFactory;
+import org.opengis.filter.LogicalOperator;
+import org.opengis.filter.LogicalOperatorName;
+import org.opengis.filter.ValueReference;
 
 import static org.apache.sis.internal.cql.CQLParser.*;
 
 
 /**
  *
  * @author  Johann Sorel (Geomatys)
- * @version 1.0
- * @since   1.0
+ * @version 1.1
+ * @since   1.1
```

Comment:
Thanks for the review! Yes, the `@since` tag should show when the file was introduced in public API. But this CQL file has never been released yet; we removed it from SIS 1.0 release because it was not ready at that time. SIS 1.1 would be its first release.

On the `@version` tag, the current practice in SIS is to set that number to the latest SIS release where that file got significant change. So even if we update a file many times between SIS 1.0 and 1.1, the `@version` tag is still updated only once until the release. Which changes are considered "significant" is left to developer appreciation. For example I do not update the version number if I only improved javadoc with no code change.

Reply:
>because it was not ready at that time. SIS 1.1 would be its first release.

Ah! :+1: makes sense!

>Which changes are considered "significant" is left to developer appreciation. For example I do not update the version number if I only improved javadoc with no code change.

Fair enough, marking this one as resolved to simplify the review for others. Thanks Martin!